src/harness.js

```diff
@@ -234,11 +235,41 @@ class Harness {
       /* this.log respects verbose option, use logFormErrors here */
       console.error(`Error encountered while filling form:`, JSON.stringify(fillResult.errors, null, 2));
     }
+    return fillResult;
+  }
 
+  /**
+   * Loads and fills a contact form,
+   *
+   * @param {string} contactType Type of contact that should be created
+   * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+   */
+  async fillContactForm(contactType, ...answers) {
```

Comment:
Consider deprecating this interface and adding a new one `fillContactCreateForm` that reuses this